resend.yaml

```diff
@@ -2127,499 +3215,580 @@ components:
-          description: Timestamp indicating when the template was published.
-          nullable: true
-        has_unpublished_versions:
-          type: boolean
-          description: Indicates whether the template has unpublished versions.
-    TemplateListItem:
+          enum:
+            - public
+            - private
+          description: The visibility of the topic on the unsubscribe page.
+    UpdateTopicResponseSuccess:
       type: object
       properties:
         id:
           type: string
-          description: The ID of the template.
-        name:
-          type: string
-          description: The name of the template.
-        status:
-          type: string
-          description: The publication status of the template.
-          enum: [draft, published]
-        published_at:
-          type: string
-          format: date-time
-          nullable: true
-          description: Timestamp indicating when the template was published.
-        created_at:
+          description: The ID of the topic.
+          example: 'b6d24b8e-af0b-4c3c-be0c-359bbd97381e'
+        object:
           type: string
-          format: date-time
-          description: Timestamp indicating when the template was created.
-        updated_at:
+          description: The object type of the response.
+          example: topic
+    RemoveTopicResponseSuccess:
+      type: object
+      properties:
+        id:
           type: string
-          format: date-time
-          description: Timestamp indicating when the template was last updated.
-        alias:
+          description: The ID of the topic.
+          example: 'b6d24b8e-af0b-4c3c-be0c-359bbd97381e'
+        object:
           type: string
-          description: The alias of the template.
-    CreateTemplateRequest:
+          description: Type of the response object.
+          example: topic
+        deleted:
+          type: boolean
+          description: Indicates whether the topic was successfully deleted.
+          example: true
+    CreateContactPropertyOptions:
       type: object
       required:
-        - name
-        - html
+        - key
+        - type
       properties:
-        name:
-          type: string
-          description: The name of the template.
-        alias:
-          type: string
-          description: The alias of the template.
-        from:
+        key:
           type: string
-          description: Sender email address. To include a friendly name, use the format "Your Name <sender@domain.com>".
-        subject:
+          description: The property key. Max length is 50 characters. Only alphanumeric characters and underscores are allowed.
+        type:
           type: string
-          description: Email subject.
-        reply_to:
-          type: array
-          items:
-            type: string
-          description: Reply-to email addresses.
-        html:
+          enum:
+            - string
+            - number
```

Comment:
Just confirming, we don't support booleans do we? The implementation still does: https://github.com/resend/resend-api/blob/1467ffa9af287dd254dcfd8d9b5f9c60cc4d710c/src/routes/contact-properties/validations/contact-property-schema.ts#L9